src/CacheItemInterface.php

```diff
@@ -72,12 +72,12 @@ public function isHit();
      * @return static
      *   The invoked object.
      */
-    public function set($value);
+    public function set(mixed $value): static;
```

Comment:
Does that mean that immutability is required?

Reply:
`CacheItem` explicitly cannot be an immutable object.  It's a builder object.  set() must return "the invoked object", so it's going to be of the same type (static) no matter what.  If it's not, it's an error.